docs/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+Fixes # .
+
+This PR proposes the following changes:
+-
+-
+
+@mention_to_someone_if_necessary
```

Comment:
```suggestion
<!-- Feel free to add more information and sections you think are relevant. -->
```

Reply:
I think both options are good, but i prefer the simpler one to keep things standard across PRs, since any discussion would probably had happened in the issue.

Reply:
Yeah I think it makes sense. Alright :)